src/components/Table/hoc/withTableSorting/withTableSorting.tsx

```diff
@@ -76,7 +76,7 @@ export function withTableSorting<I extends TableDataItem, E extends {} = {}>(
             const {data, columns} = this.props;
             const sortState = this.getSortState();
 
-            if (this.isControlledState() || sortState.length === 0) {
+            if (sortState.length === 0) {
```

Comment:
It's breaking change. I think it's better add prop that controls a use of inner/external sorting,  and `onSortStateChange` should be always called, not only in controlled state.

```ts
const {useExternalSortings = this.isControlledState()} = this.props
if (useExternalSorting || sortState.length === 0) {
```